### ISSUE TITLE
Delete gchp/v1.0.0 branch

### ISSUE THREAD

**Issue by tclune** [GEOS-ESM/MAPL#13]
Once the corresponding tag is verified, this branch does not need to persist.

**Comment by lizziel** [comment]
I forked the MAPL repo to the GEOS-Chem GitHub organization and we are starting to test building it with cmake in the new GCHP repo. To start we'll just be building the GMAO libraries since it will be incompatible with GCHP which uses a newer MAPL and needs custom updates beyond tag gchp/v1.0.0. 

Do you have an idea of when the legacy develop branch updates (as of when NASA GitHub went offline) will be available in master?

**Comment by tclune** [comment]
Adding @mathomp4, since he has boots on the ground and can comment more accurately about this.   The unanticipated bit is we needed to create MAPL specific branches for the gridcomps due to some ESMF changes in order to just test the MAPL develop stuff.  

The git transition is still creating a lot of extra work for my group, and we're down both Ben and myself for this week.   So please be patient.

**Comment by lizziel** [comment]
Thanks Tom and Matt. No hurry from my end since we aren't planning to release the next big version update until after ESMF v8 is released, which puts us to Sept at the earliest.

**Comment by mathomp4** [comment]
@lizziel It will probably be a while until the old developer-develop gets into `master` here. I know @bena-nasa was trying to compile/build the developer-develop yesterday and I might try to help when I have a chance. Then, we have to show we can get the same answer as developer-develop.

After that, because developer-develop is technically non-zero-diff to `master` (due to use of ESMF regridding) it will trigger a *lot* of testing on this end. GMAO will need to validate it with 30-year EMIPs and the like to prove that it is the same climate.

Though, I suppose, once we prove to ourselves that it's working, there is a train of thought to say MAPL 2.0 can be put on `master` itself in this repo. The older versions will still be here and can always have bugfixes in some sort of `bugfix-v1` branch or whatever.

**Comment by lizziel** [comment]
Thanks @mathomp4. Having a dev branch works as well, and I see that is there now with the base code I developed off of. We are actually using 5 repos: MAPL, GMAO_Shared, ESMA_cmake, ecbuild, and GEOSgcm_GridComp. Is there a repo that shows the submodule SHAs for these used together, specifically using MAPL develop? I am essentially looking for where to check version compatibility now, or in the future if a repo pulling them together is not there yet.

**Comment by mathomp4** [comment]
@lizziel Not yet, but yesterday, @bena-nasa did show that he can duplicate the results of developer-develop with code changes in Github. But, I'd like to have him around (say Monday) before I officially pull in a lot of his changes. Plus a few are in MAPL and I'm not that qualified to approve those. I mean, I'm fairly sure their correct because, well, they were needed, but still...

That said, hopefully by mid next week, I can get you tags of all five repos. Between you and @tclune, you could come up with a tag name that works (`gchp/v1.0.0` could have a nice ring :smile: ) and I could easily tag the corresponding commits with that. Then your scripting is simple.

**Comment by lizziel** [comment]
@mathomp4 Thanks! To be clear, I want to update to the latest MAPL and not necessarily stick with gchp/v1.0.0. I noticed folks are starting to use GEOSctm and I found the version of MAPL it uses, 1.1.3, along with the mapping to other repo versions. I diffed 1.1.3 with what we are using and it appears to be further along that our version, which is a good sign. I'm thinking we can start out with the versions GEOSctm uses and then update from there as needed. I'm not going to run GCHP with it yet, we just need to be able to test building with our new setup and ideally not using any branches that will be deleted and never merged in (gchp/v1.0.0).
